css/style.css

```diff
@@ -332,6 +334,391 @@ body {
   background: #2c2c2c;
 }
 
+/* Второй блок */
+
+.video {
+  min-height: 494px;
+  min-width: 1030px;
+  background: #000;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 77px;
+}
+
+.video__frame {
+  background: url(../img/video-bg.png) no-repeat;
+  height: 361px;
+  width: 644px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 12px;
+}
+
+.video__btn {
+  height: 47px;
+  width: 47px;
+  background: #fff;
+  border-radius: 50%;
+  box-shadow: 0 0 0px 4px #ffffff78;
+  border: none;
+  position: relative;
+  outline: none;
+}
+
+.video__btn::before {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 16px;
+  bottom: 15px;
+  background: url('../icons/Rectangle.png');
+  width: 16px;
+  height: 16px;
+}
+
+.video__btn:hover {
+  cursor: pointer;
+  box-shadow: inset 0 0 0px 2px #5452524a;
+}
+
+.video__btn:active {
+  cursor: pointer;
+  box-shadow: inset 0 0 0px 2px #ffffff00;
+}
+
+.video__text-wrapper {
+  width: 266px;
+  height: 259px;
+  margin-right: 22px;
+  margin-top: 70px;
+}
+
+.video__header {
+  font-size: 22px;
+  line-height: 27px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #fff;
+  margin-bottom: 15px;
+}
+
+.video__text {
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 100;
+  color: #fff;
+  margin-bottom: 17px;
+}
+
+.video__link {
+  display: inline-block;
+  width: 140px;
+  text-decoration: none;
+  font-size: 12px;
+  line-height: 11px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #fff;
+  position: relative;
+}
+
+.video__link::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 93px;
+  bottom: 2px;
+  background: url('../icons/arrow.png');
+  width: 45px;
+  height: 8px;
+}
+
+.video__link:hover {
+  color: #b5b5b5;
+}
+
+.video__link:active {
+  color: #fff;
+  -webkit-text-stroke: 1px rgba(0, 0, 0, .5);
+  -webkit-text-fill-color: rgba(255, 255, 255, .4);
+}
+
+.video__link:hover::after{
+  opacity: .4;
+}
+
+/* Третий блок */
+
+.stories__card-wrapper {
+  margin-bottom: 78px;
+}
+
+.stories__header {
+  font-size: 22px;
+  line-height: 21px;
+  text-transform: uppercase;
+  font-weight: 700;
+  color: #000;
+  text-align: center;
+  margin-bottom: 44px;
+}
+
+.stories__card-wrapper {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stories-card {
+  max-width: 307px;
+}
+
+.stories-card__img {
+  margin-bottom: 29px;
+}
+
+.stories-card__img:hover {
+  opacity: .4;
+  cursor: pointer;
+}
+
+.stories-card__img_1024 , .stories-card__img_768 {
+  display: none;
+}
+
+.stories-card__header {
+  font-size: 18px;
+  line-height: 17px;
+  font-weight: 700;
+  color: #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)